typescript/packages/common-memory/provider.ts

```diff
@@ -108,10 +108,10 @@ class MemoryProviderSession implements ProviderSession<Protocol>, Subscriber {
         await this.invoke(command);
       },
       abort: async () => {
-        await this.close();
+        this.close();
       },
       close: async () => {
-        await this.close();
+        this.close();
```

Comment:
This would swallow the error while prior version will propagate it.

Reply:
you could do

```
    abort: () => this.close();
    close: () => this.close();
```

Reply:
assuming whatever is calling this is doing an await